styles/mandys.css

```
@@ -0,0 +1,3 @@
+.section-Maria { 
+    color: lightseagreen;
+}
```

Aplicar salto de línea al final del archivo.

```suggestion
.section-Maria { 
    color: lightseagreen;
}

```